meetings/2018-01-17/minutes.md

```diff
@@ -0,0 +1,417 @@
+information in the case of errors, but the editor also needs support for
+building. Currently sbt has to be spun up in a separate terminal. He proposes a
+protocol where *n* editors can talk to *m* build tools, but would start with
+just a few.
+
+Martin suggests that the presentation compiler could have configuration to
+start up any build tool, possibly with an overridable favourite, and then talk
+to that build tool through a common BSP, which would share the classpath and
+other configuration.
+
+He proposes starting with sbt, which already has an LSP implementation, but
+inviting other build tools such as Bloop to use the proposal.
+
+TASTY is a way for the compiler to serialize its abstact syntax trees. Any file
+that Dotty compiles will produce a TASTY tree. Any TASTY trees from files that
+have been built will be available to clients, quickly, whereas any files which
+have not been built successfully would need to have that information provided
+by the compiler by some other means. Martin is concerned about the
+out-of-the-box experience for newcomers, and thinks that it's essential that
+feedback is provided early for these users.
+
+Martin is extremely excited by all of the development, and hopes that all the
+efforts can come together to produce something really great!
+
+## Compilers
+
+Jon introduces the next section of the meeting by observing that there are
+different Scala compilers for variants of the Scala language, and that now is
+the best time to be considering support for Scala compilers other than Scalac,
+rather than trying to retrofit support at a later date, even though the vast
+majority of users will be using Scalac 2.x.
+
+He sees a strong desire from other Scala compiler developers to have good
+tooling, and with that a strong willingness to contribute to efforts like this,
+so we should do everything possible to work together.
+
+### Scalac - Adriaan Moors, Lightbend
+
+Adriaan reiterates that tooling will be key to the next wave of Scala adoption,
+and wants to encourage the current efforts.
+
+Scalac originated as a batch compiler, and adapted into an incremental
+compiler. Dependencies for fine-grained compilation are different for Scala
+than for, say, Java. TASTY and SemanticDB are useful for exchanging that
+information. He's very happy to help enable the development of these features
+for Scalac.
+
+### Rsc - Eugene Burmako, Twitter
+
+[Rsc](http://goo.gl/LHmuym) is an experiment aiming to get 5-10x compilation
+speedups. It currently only supports a subset of Scala, and provides partial
+compilation, but is already showing impressive performance.
+
+Eugene believes that it will be possible to typecheck methods in parallel
+during compilation, and notes that separating name resolution and typechecking
+allows is essential for a good user experience. Rsc cross-compiles to Scala
+Native. Eugene spoke briefly about some future plans for Rsc, described in more
+detail in his slides.
```

Comment:
Maybe let's move the link to the slides from the beginning of the section to "in his slides"?